### ISSUE TITLE
Parameter substitution only seems to work for one parameter

### ISSUE THREAD

**Issue by unikitty37** [victor-am/rails-ranger#26]
I'm having trouble getting the parameter substitution to work with more than one parameter:

```js
api.get('/first/:a/second/:b/third/:c', { a: 'A', b: 'B', c: 'C', d: 'D' }).then((r) => {
  console.log(r)
}).catch((e) => {
  console.log(e)
})
```

should perform a GET to `/first/A/second/B/third/C?d=D`, but instead tries to GET `/first/:a/second/:b/third/C?d=D`

I've managed to work around this by using `api.get(`\``first/${a}/second/${b}/third/:c`\``)` for now, but it doesn't look very clean (just like attempting to put backticks in a code block in GitHub 🙄 )

I can't see anything in `_paramsToPath` that could be causing this — do you know what could be going wrong here?

(I did try replicating this with the RouteBuilder, but the import in the README seems to be wrong:

```js
import { RouteBuilder } from RailsRanger
const routes = new RouteBuilder
```

produces `Uncaught Error: Module build failed: SyntaxError: Unexpected token`. Replacing the import with

```js
import { RouteBuilder } from 'rails-ranger'
```

produces `TypeError: __WEBPACK_IMPORTED_MODULE_3_rails_ranger__.RouteBuilder is not a constructor`. What *is* the correct way to import the RouteBuilder?)

**Comment by victor-am** [comment]
Thanks a lot for the bug report ✨ 

I've opened a fix on the #27 PR, can you take a look if it makes sense in solving your problem?
I should merge this and release a new patch version this early this week 👍

**Comment by unikitty37** [comment]
Thanks! It looks good for this case, but doesn't `replace` only work on the first instance?

`api.get('/some_path/:foo/:bar/:foo', { foo: 1, bar: 2 })` would give `/some_path/1/2/:foo` still — would this be better?

```js
processedPath = processedPath.split(symbol).join(params[key])
```

**Comment by victor-am** [comment]
Nice catch 😄 
It should be fixed on 70e8d7ea25670b646f8a2c5a75119295bfcd35d0

**Comment by victor-am** [comment]
Thanks again for the bug report, it should be fixed in the new [1.1.2 version](https://github.com/victor-am/rails-ranger/releases/tag/1.1.2) I've just published 😄

**Comment by unikitty37** [comment]
Thanks — all working now at this end 🎉
